spec/mailers/previews/webhook_mailer_preview.rb

```diff
@@ -1,9 +0,0 @@
-# frozen_string_literal: true
-
-class WebhookMailerPreview < BasePreviewMailer
```

Comment:
For my curiousity, how do we use this ?

Reply:
Neat! I didn't know about that. I guess I'm more used to working with Sendgrid, Mailchimp, etc. than Rails mailer.